### PR TITLE
WEBAPP-477: Use adaptive image quality for events

### PR DESCRIPTION
**Pull request by Schedulaar**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/common/components/Page.js [hunk 1]
```diff
@@ -7,18 +7,21 @@ import Caption from './Caption'
...
+  height: ${THUMBNAIL_WIDTH}px;
   margin: 10px auto;
   padding-bottom: 10px;
   object-fit: contain;
 `
 
 type PropsType = {|
   title: string,
-  thumbnail?: string,
+  defaultThumbnailSrc?: string, // necessary for IE11 support
```
Comment (klinzo) [inline src/modules/common/components/Page.js:23]: :vomiting_face:

File: src/modules/common/utils/featuredImageToSrcSet.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { FeaturedImageModel } from '@integreat-app/integreat-api-client'
+
+const ten = 10
```
Comment (klinzo) [inline src/modules/common/utils/featuredImageToSrcSet.js:5]: :laughing:

File: src/modules/common/utils/featuredImageToSrcSet.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const roundToOneDecimal = number => Math.round(number * ten) / ten
+
+/**
+ * This converts a FeaturedImageModel to a scrset.
```
Comment (klinzo) [inline src/modules/common/utils/featuredImageToSrcSet.js:9]: Maybe add a link to information about srcsets.

File: src/routes/pois/containers/PoisPage.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ export class PoisPage extends React.Component<PropsType> {
 
       if (poi) {
         return (
-          <Page thumbnail={poi.thumbnail}
+          <Page defaultThumbnailSrc={poi.thumbnail}
```
Comment (klinzo) [inline src/routes/pois/containers/PoisPage.js:42]: No such thing as different resolutions for pois?

File: src/modules/common/utils/featuredImageToSrcSet.js (hunk 3 above)
Comment (Schedulaar) [reply src/modules/common/utils/featuredImageToSrcSet.js:9]: Hmm... as this is a standard-attribute, I think we should not add a link...  I'll fix the typo though :sweat_smile:

Comment (Schedulaar) [reply src/modules/common/utils/featuredImageToSrcSet.js:9]: Hmm... as this is a standard-attribute, I think we should not add a link...  I'll fix the typo though :D

File: src/routes/pois/containers/PoisPage.js (hunk 4 above)
Comment (Schedulaar) [reply src/routes/pois/containers/PoisPage.js:42]: No, not yet... We should add one though... Can you create an issue for the cms?

Comment (klinzo) [reply src/routes/pois/containers/PoisPage.js:42]: Could you do that please?

Comment (klinzo) [reply src/routes/pois/containers/PoisPage.js:42]: Not sure if it is a issue for the current cms at all

Comment (Schedulaar) [reply src/routes/pois/containers/PoisPage.js:42]: I created an issue: https://issues.integreat-app.de/browse/WP-6

**Pull request merged**